simulariumio/tests/test_agents_from_lists.py

```diff
@@ -0,0 +1,493 @@
+import numpy as np
+
+from simulariumio import AgentData
+
+
+def test_agent_data_jagged_list():
+    list_data = {
```

Comment:
It would be nice to define a data class for this instead of using a dict, like `AgentDataLists` or something, so users don't have to guess the key names exactly and so the list dimensions are documented with the types

Reply:
- [ ] create AgentDataList class